OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -21,10 +22,11 @@ namespace OpenRA.Mods.Common.Widgets
 	public class SupportPowerTimerWidget : Widget
 	{
 		public readonly string Font = "Bold";
-		public readonly string Format = "{0}: {1}";
+		public readonly string Format = "{0}'s {1}: {2}";
```

Comment:
Actually, the ideal way would be to let this go through localization because `Matt's Atom Bomb` becomes `Matts Atombombe` in German, for example.

Reply:
I agree, but then it couldn't be merged into prep if that's a concern